docs/pages/branding/about.js

```diff
@@ -0,0 +1,1067 @@
+          direction="row"
+          spacing={1}
+        >
+          <Grid item>
+            <Typography variant="h4">{title}</Typography>
+          </Grid>
+          <Grid item>
+            <ArrowCirleIcon />
+          </Grid>
+        </Grid>
+        <Typography className="DiscoverMoreCard-content">{children}</Typography>
+        <Box
+          component="img"
+          alt={alt}
+          src={src}
+          sx={{
+            height: 250,
+            width: styleProps.imagePosition === 'center' ? 280 : 330,
+            display: 'block',
+            marginTop: 'auto',
+          }}
+        />
+      </CardContent>
+    </Card>
+  );
+})(({ theme, styleProps = {} }) => ({
+  '&[class*="MuiCard-root"]': {
+    background: theme.palette.secondary.main,
+    color: 'white',
+  },
+  '& img': {
+    marginLeft:
+      styleProps.imagePosition === 'center' || styleProps.imagePosition === 'end' ? 'auto' : 0,
+    marginRight: styleProps.imagePosition === 'center' ? 'auto' : 0,
+  },
+  '& svg': {
+    marginTop: 4,
+  },
+  height: theme.spacing(50),
+  padding: 0,
+  '& .DiscoverMoreCard-header': {
+    textDecoration: 'none',
+    color: 'white',
+    padding: theme.spacing(2),
+  },
+  '& .DiscoverMoreCard-content': {
+    padding: theme.spacing(2),
+  },
+  '& [class*="MuiCardContent-root"]': {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    padding: 0,
+    '&:last-child': {
+      paddingBottom: 0,
+    },
+  },
+}));
+
+export default function About() {
+  return (
+    <Page>
+      <AboutUsRoot>
+        <Grid container className="MuiGrid-mission">
+          <Grid container className="MuiGrid-centered MuiGrid-bottomGutter" alignItems="center">
+            <Grid item>
+              <Typography variant="h1" align="center" gutterBottom>
+                We&apos;re on a mission to make building UIs with{' '}
+                <UnderlinedText>React</UnderlinedText> fun.
+              </Typography>
+            </Grid>
+            <Grid item className="MuiGrid-centered-content">
+              <Typography align="center" gutterBottom>
+                Material-UI started back in 2014 to unify{' '}
+                <Link href="https://reactjs.org/">React</Link> and{' '}
+                <Link href="https://material.io/design">Material Design</Link>.
+              </Typography>
+              <Typography align="center">
+                Today, Material-UI has grown to become one of the world&apos;s most popular React
+                libraries – used by a vibrant community of more than <b>2M developers</b> in over{' '}
+                <b>180 countries</b>.
+              </Typography>
+            </Grid>
+          </Grid>
+
+          <Grid container spacing={1} className="MuiGrid-bottomGutter MuiGrid-relative">
+            <Grid item lg={6} xs={12} className="MuiGrid-relative">
+              <Box
+                component="img"
+                src="/static/images/branding/AboutUs-1.jpg"
```

Comment:
I think that:

- we should isolate the images per page, to make them easier to delete in the future, once we revamp the page, maybe in 2 years.
- it's best to stick to a single filename convention, so far, kebab-case.
- we can flatten the import structure.

```suggestion
                src="/static/about/top-left.jpg"
```

Reply:
Done